app/main/views/direct_award.py

```diff
@@ -141,6 +143,45 @@ def get_project(project_external_id):
     return single_result_response("project", project, serialize_kwargs={"with_users": True}), 200
 
 
+@main.route('/direct-award/projects/<int:project_external_id>', methods=['PATCH'])
+def update_project(project_external_id):
+    uniform_now = datetime.datetime.utcnow()
```

Comment:
I'm not sure I understand what you mean exactly, so if you could create the tech debt ticket that would be appreciated.

Reply:
You'll see it in most things I write that use a notion of "current time". Haven't gone as far as pulling it from the db because most the time I do this kind of thing outside the api. But then again, i'm not sure we really currently use db time at all - all our "default now"s use `datetime` don't they?